core/src/main/python/compare_model.py

```diff
@@ -254,20 +260,11 @@ def main():
 
         for f in [model1, model2]:
             if not os.path.exists(f):
-                raise CLAException("Model %s does not exists" % f)
+                raise CLAException(2, 'WLSDPLY-85717', [f])
```

Comment:
Can this use ExitCode.ERROR ? (recently merged to main).
3 places in this file.

Reply:
Sure.